README.md

```diff
@@ -19,6 +19,10 @@ Execute it:
 npx codecept-ui
 ```
 
+# ENV
+
+* port: Update default port of websocket (3000) and Codecept UI (defaultPort + 1)
```

Comment:
Could you add also an example running codeceptjs with this env variable enabled?

Reply:
Done :)